Assets/Scripts/Game/UI/HealthDebugText.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+
+public class HealthDebugText : MonoBehaviour
+{
+    private TextMeshProUGUI text;
```

Comment:
I'm not saying this is bad, but wouldn't it be easier to use `OnGUI` for debug text?

Reply:
Yeah, probably. I didn't even really think about it TBH. For this PR I won't change it, but I'll definitely look into doing it that way in the future for efficiency.